packages/Button/src/Button.js

```diff
@@ -154,13 +155,17 @@ const Button = React.forwardRef((props, ref) => {
   }
 
   const buttonProps = {
+    tabIndex,
+    "data-pka-anchor": "button",
+    "aria-pressed": isActive,
+    ...moreProps,
+    "data-has-forced-focus": hasForcedFocus || null,
+    isActive,
```

Comment:
Hmm why did `isActive` get pulled out of moreProps (line 103), only to be added back in here?

Reply:
Oh, I need it for `aria-pressed` as well (above on L160).

Reply:
ah, didnt see that sneaky line in there